icons/src/scripts/convert.js

```diff
@@ -0,0 +1,79 @@
+const fs = require("fs");
+const path = require("path");
+const svgr = require("@svgr/core").default;
+const camelcase = require("camelcase");
+const { formatComponentName } = require("./formatComponentName");
+const { svgo } = require("./convertUtils/setupSvgo");
+const SVG_PATH = path.resolve(__dirname, "..", "svgs");
+const COMPONENT_PATH = path.resolve(__dirname, "..", "..");
+
+(async () => {
+  // Move all the files in corresponding component directories
+  return fs
+    .readdirSync(SVG_PATH)
+    .filter(svgPathFilename =>
+      fs.lstatSync(path.join(SVG_PATH, svgPathFilename)).isDirectory()
+    )
+    .map(async svgPathDirectory => {
+      const fullSvgDirectory = path.join(SVG_PATH, svgPathDirectory);
+
+      return Promise.all(
+        fs
+          .readdirSync(fullSvgDirectory)
+          .filter(filename => path.extname(filename) === ".svg")
+          .map(async filename => {
+            const svgCode = fs.readFileSync(
+              path.join(fullSvgDirectory, filename),
+              "utf-8"
+            );
+
+            // We have to use a custom svgo setup because the `svgr` one isn't
+            // configurable and will sometimes remove fills and strokes that we
+            // don't want removed.
+            const { data: svg } = await svgo.optimize(svgCode, {
```

Comment:
Could I confirm that this issue is resolved by this custom svgo config?  
https://github.com/mdg-private/engine-frontend/pull/1513#discussion_r296054605

Reply:
Great find @cheapsteak . Check out the build preview in Netlify for `IconServices`:

![image](https://user-images.githubusercontent.com/5224118/60030020-40d95800-9670-11e9-95a4-8e16b9e0183e.png)